# Задача №8 — Аудит ИТ систем ВУЗа для решения вопроса о переходе на ОС «Альт»

In [2]:
import pandas as pd
import numpy as np
from tabulate import tabulate

In [3]:
data = pd.read_csv(r'../data/0.1_uni_data_sites_joined_distinct_prepared_filled.csv')

In [4]:
data.tail()

ShortName  \
764                                      Академия ВЭГУ   
765  ЧОУ ВО «Казанский инновационный университет им...   
766  ЧОУ ВО «Санкт-Петербургский университет технол...   
767  ЧОУ ВО «Сибирский юридический университет», Си...   
768                               Университет «Реавиз»   

                           RegionName  \
764           Республика Башкортостан   
765  Республика Татарстан (Татарстан)   
766                г. Санкт-Петербург   
767                    Омская область   
768                г. Санкт-Петербург   

                                              FullName                WebSite  
764  частное образовательное учреждение высшего обр...   https://www.vegu.ru/  
765  частное образовательное учреждение высшего обр...       https://ieml.ru/  
766  частное образовательное учреждение высшего обр...     https://spbume.ru/  
767  частное образовательное учреждение высшего обр...  https://www.siblu.ru/  
768  частное учреждение образовательная организация...  https://spbreaviz.ru/

In [5]:
data.describe()

ShortName RegionName  \
count              746        766   
unique             742         87   
top     ФГБОУ ВО «БГУ»  г. Москва   
freq                 2        149   

                                                 FullName  \
count                                                 769   
unique                                                769   
top     частное учреждение образовательная организация...   
freq                                                    1   

                           WebSite  
count                          756  
unique                         747  
top     http://www.academy.fsb.ru/  
freq                             3

Проверяю, есть ли пропуски в данных

In [6]:
data.isnull().sum()

ShortName     23
RegionName     3
FullName       0
WebSite       13
dtype: int64

In [7]:
data_wo_site = data.loc[pd.isna(data['WebSite'])]

In [8]:
data_wo_site.shape

(13, 4)

In [9]:
(data_wo_site.shape[0] / data.shape[0]) * 100

1.6905071521456438

13 ВУЗов без сайтов -- 1.69%. Можно найти и восполнить данные руками

In [10]:
print(tabulate(data_wo_site[['FullName']], headers='keys', tablefmt='plain'))

     FullName
 85  Государственное образовательное учреждение «Приднестровский государственный университет им. Т.Г. Шевченко»
104  Негосударственное частное образовательное учреждение высшего образования «Северный институт предпринимательства»
123  Религиозная духовная образовательная  организация высшего образования  «Заокский университет Церкви Христиан-Адвентистов Седьмого Дня»
162  ФЕДЕРАЛЬНОЕ ГОСУДАРСТВЕННОЕ КАЗЕННОЕ ВОЕННОЕ ОБРАЗОВАТЕЛЬНОЕ УЧРЕЖДЕНИЕ ВЫСШЕГО ОБРАЗОВАНИЯ «ДОНЕЦКОЕ ВЫСШЕЕ ОБЩЕВОЙСКОВОЕ КОМАНДНОЕ УЧИЛИЩЕ» МИНИСТЕРСТВА ОБОРОНЫ РОССИЙСКОЙ ФЕДЕРАЦИИ
194  Федеральное государственное бюджетное образовательное учреждение высшего образования «Уральский государственный университет путей сообщения»
196  Федеральное государственное бюджетное образовательное учреждение высшего образования «Херсонский аграрный университет»
202  Федеральное государственное казенное образовательное учреждение высшего образования «Институт Федеральной службы безопасности Российской Федерации (г. Н

In [11]:
data['WebSite'].iloc[85] = 'http://spsu.ru/'
data['WebSite'].iloc[123] = 'https://zda.zau.ru/'
data['WebSite'].iloc[252] = 'https://pc.ipc39.ru/'
data['WebSite'].iloc[260] = 'http://bru.by/'
data['WebSite'].iloc[262] = 'https://ktlt.ru'
data['WebSite'].iloc[307] = 'https://urfu.ru/'

Сразу проверяю

In [12]:
data.loc[data_wo_site.index]

ShortName  \
85                                                 NaN   
104    НЧОУ ВО «Северный институт предпринимательства»   
123                                                NaN   
162  «ДОНЕЦКОЕ ВЫСШЕЕ ОБЩЕВОЙСКОВОЕ КОМАНДНОЕ УЧИЛИЩЕ»   
194  ФГБОУ ВО «Уральский государственный университе...   
196                                     ФГБОУ ВО "ХАУ"   
202     ФГКОУ Институт ФСБ России (г. Нижний Новгород)   
206                                      ЧОУ ВО «СаГА»   
249                                              СОГПИ   
252               ГБОУ ВО КО «Педагогический институт»   
260                  Белорусско-Российский университет   
262              МБОУ ВО «Тольяттинская консерватория»   
307  ФГАОУ ВО «УрФУ имени первого Президента России...   

                                            RegionName  \
85   образовательные учреждения, находящиеся за пре...   
104                              Архангельская область   
123                                   Тульская область   
162                       Донецкая Народная Республика   
194                               Свердловская область   
196                                 Херсонская область   
202                              Нижегородская область   
206                                  Самарская область   
249                Республика Северная Осетия - Алания   
252                            Калининградская область   
260  образовательные учреждения, находящиеся за пре...   
262                                  Самарская область   
307                               Свердловская область   

                                              FullName               WebSite  
85   Государственное образовательное учреждение «Пр...       http://spsu.ru/  
104  Негосударственное частное образовательное учре...                   NaN  
123  Религиозная духовная образовательная  организа...   https://zda.zau.ru/  
162  ФЕДЕРАЛЬНОЕ ГОСУДАРСТВЕННОЕ КАЗЕННОЕ ВОЕННОЕ О...                   NaN  
194  Федеральное государственное бюджетное образова...                   NaN  
196  Федеральное государственное бюджетное образова...                   NaN  
202  Федеральное государственное казенное образоват...                   NaN  
206  Частное образовательное учреждение высшего обр...                   NaN  
249  государственное бюджетное образовательное учре...                   NaN  
252  государственное бюджетное образовательное учре...  https://pc.ipc39.ru/  
260  межгосударственное образовательное учреждение ...        http://bru.by/  
262  муниципальное бюджетное образовательное учрежд...       https://ktlt.ru  
307  федеральное государственное автономное образов...      https://urfu.ru/

In [13]:
data.describe()

ShortName RegionName  \
count              746        766   
unique             742         87   
top     ФГБОУ ВО «БГУ»  г. Москва   
freq                 2        149   

                                                 FullName  \
count                                                 769   
unique                                                769   
top     частное учреждение образовательная организация...   
freq                                                    1   

                           WebSite  
count                          762  
unique                         753  
top     http://www.academy.fsb.ru/  
freq                             3

In [14]:
(data.loc[pd.isna(data['WebSite'])].shape[0] / data.shape[0]) * 100

0.9102730819245773

In [15]:
wo_null_site_data = data.dropna(subset=['WebSite'])

In [16]:
duplicate_sites = wo_null_site_data[wo_null_site_data.duplicated(subset=['WebSite'], keep=False)]

In [17]:
duplicate_sites.sort_values('WebSite')

ShortName  \
69                                       ГОУ ВПО ДОННУ   
407  ФГБОУ ВО «ДонГУ», Донецкий государственный уни...   
732                               ФГКОУ ГПИ ФСБ России   
739                              ФГКОУ КаПИ ФСБ России   
748                                      ПА ФСБ России   
734        ФГКОУ Институт ФСБ России (г. Екатеринбург)   
725                                Академия ФСБ России   
201                                      ФГБПОУ "БЦБК"   
354                                    ФГБОУ ВО «БрГУ»   
63                      ГОО ВПО ДОННМУ ИМ. М. ГОРЬКОГО   
406                   ФГБОУ ВО ДонГМУ Минздрава России   
159                                    ФГБОУ ВО «ЛГПУ»   
67                                   ГОУ ВО ЛНР «ЛГПУ»   
524                                ФГБОУ ВО «ОрелГУЭТ»   
523  ОГУ имени И.С. Тургенева, ФГБОУ ВО «Орловский ...   
700  Военная академия Министерства обороны Российск...   
709                         ФГКУ «Военный университет»   

                                            RegionName  \
69   образовательные учреждения, находящиеся за пре...   
407                       Донецкая Народная Республика   
732                                 Московская область   
739                            Калининградская область   
748                                          г. Москва   
734                               Свердловская область   
725                                          г. Москва   
201                                  Иркутская область   
354                                  Иркутская область   
63   образовательные учреждения, находящиеся за пре...   
406                       Донецкая Народная Республика   
159                      Луганская Народная Республика   
67   образовательные учреждения, находящиеся за пре...   
524                                  Орловская область   
523                                  Орловская область   
700                                          г. Москва   
709                                          г. Москва   

                                              FullName  \
69   ГОСУДАРСТВЕННОЕ ОБРАЗОВАТЕЛЬНОЕ УЧРЕЖДЕНИЕ ВЫС...   
407  федеральное государственное бюджетное образова...   
732  федеральное государственное казенное образоват...   
739  федеральное государственное казенное образоват...   
748  федеральное государственное казенное образоват...   
734  федеральное государственное казенное образоват...   
725  федеральное государственное казенное образоват...   
201  Федеральное государственное бюджетное професси...   
354  федеральное государственное бюджетное образова...   
63   ГОСУДАРСТВЕННАЯ ОБРАЗОВАТЕЛЬНАЯ ОРГАНИЗАЦИЯ ВЫ...   
406  федеральное государственное бюджетное образова...   
159  ФЕДЕРАЛЬНОЕ ГОСУДАРСТВЕННОЕ БЮДЖЕТНОЕ ОБРАЗОВА...   
67   ГОСУДАРСТВЕННОЕ ОБРАЗОВАТЕЛЬНОЕ УЧРЕЖДЕНИЕ ВЫС...   
524  федеральное государственное бюджетное образова...   
523  федеральное государственное бюджетное образова...   
700  федеральное государственное казенное военное о...   
709  федеральное государственное казенное военное о...   

                        WebSite  
69             http://donnu.ru/  
407            http://donnu.ru/  
732          http://gpi.fsb.ru/  
739          http://gpi.fsb.ru/  
748  http://www.academy.fsb.ru/  
734  http://www.academy.fsb.ru/  
725  http://www.academy.fsb.ru/  
201           https://brstu.ru/  
354           https://brstu.ru/  
63             https://dnmu.ru/  
406            https://dnmu.ru/  
159           https://lgpu.org/  
67            https://lgpu.org/  
524      https://oreluniver.ru/  
523      https://oreluniver.ru/  
700        https://vumo.mil.ru/  
709        https://vumo.mil.ru/

Дубликаты:
10 июня 2023 года подписано распоряжение Правительства Российской Федерации №1523-р о реорганизации ФГБОУ ВО «БрГУ» в форме выделения из него федерального государственного бюджетного профессионального образовательного учреждения «Братский целлюлозно-бумажный колледж».
"За пределами РФ из ЛНР и ДНР"

In [18]:
index_to_drop = [69, 739, 748, 734, 201, 63, 67, 524, 700]

data_cleansed = wo_null_site_data.drop(index_to_drop)

In [19]:
data_cleansed.describe()

ShortName RegionName  \
count               730        750   
unique              726         87   
top     ФГБОУ ВО «НГПУ»  г. Москва   
freq                  2        147   

                                                 FullName  \
count                                                 753   
unique                                                753   
top     частное учреждение образовательная организация...   
freq                                                    1   

                      WebSite  
count                     753  
unique                    753  
top     https://spbreaviz.ru/  
freq                        1

Экспортирую полученный DataFrame для дальнейшей доработки

In [20]:
data_cleansed.to_csv('../data/1.0_uni_data_cleared.csv')

In [21]:
data_cleansed[data_cleansed['WebSite'] == 'https://swsu.ru/']

ShortName       RegionName  \
679  ФГБОУ ВО «Юго-Западный государственный универс...  Курская область   

                                              FullName           WebSite  
679  федеральное государственное бюджетное образова...  https://swsu.ru/

In [22]:
batch = data_cleansed.iloc[0:10]

In [23]:
batch

ShortName               RegionName  \
0                           РУАН ИМЕНИ Г.Б. МИРЗОЕВА                г. Москва   
1             АНО ВО «НАЦИОНАЛЬНЫЙ ИНСТИТУТ ДИЗАЙНА»                г. Москва   
2  АНОО ВО «Воронежский экономико-правовой инстит...      Воронежская область   
3                                     АНООВО «ЕУСПб»       г. Санкт-Петербург   
4                                       АНООВО «КИУ»  Калининградская область   
5     Международный институт компьютерных технологий      Воронежская область   
6                                      АНООВО «ПССИ»       Московская область   
7                                    АНОО ВО "СИБИТ"           Омская область   
8            Сколковский институт науки и технологий       Московская область   
9                  Российский университет кооперации       Московская область   

                                            FullName  \
0  АВТОНОМНАЯ НЕКОММЕРЧЕСКАЯ ОРГАНИЗАЦИЯ ВЫСШЕГО ...   
1  АВТОНОМНАЯ НЕКОММЕРЧЕСКАЯ ОРГАНИЗАЦИЯ ВЫСШЕГО ...   
2  Автономная некоммерческая образовательная орга...   
3  Автономная некоммерческая образовательная орга...   
4  Автономная некоммерческая образовательная орга...   
5  Автономная некоммерческая образовательная орга...   
6  Автономная некоммерческая образовательная орга...   
7  Автономная некоммерческая образовательная орга...   
8  Автономная некоммерческая образовательная орга...   
9  Автономная некоммерческая образовательная орга...   

                       WebSite  
0              https://raa.ru/  
1      https://nid-design.org/  
2             https://vepi.ru/  
3            https://eusp.org/  
4            https://kiu39.ru/  
5         https://www.iict.ru/  
6  https://www.podolsk-ssi.ru/  
7       https://sibit.sano.ru/  
8      http://www.skoltech.ru/  
9          https://www.ruc.su/

In [24]:
batch.to_csv('../data/batches/0.1_testing_batch.csv', index=False)

In [25]:
data_w_soft = pd.read_csv('../data/1.4_llama3_soft_cleared.csv')

In [26]:
data_w_soft.head()

Unnamed: 0                                          ShortName  \
0           0                           РУАН ИМЕНИ Г.Б. МИРЗОЕВА   
1           1             АНО ВО «НАЦИОНАЛЬНЫЙ ИНСТИТУТ ДИЗАЙНА»   
2           2  АНОО ВО «Воронежский экономико-правовой инстит...   
3           3                                     АНООВО «ЕУСПб»   
4           4                                       АНООВО «КИУ»   

                RegionName                                           FullName  \
0                г. Москва  АВТОНОМНАЯ НЕКОММЕРЧЕСКАЯ ОРГАНИЗАЦИЯ ВЫСШЕГО ...   
1                г. Москва  АВТОНОМНАЯ НЕКОММЕРЧЕСКАЯ ОРГАНИЗАЦИЯ ВЫСШЕГО ...   
2      Воронежская область  Автономная некоммерческая образовательная орга...   
3       г. Санкт-Петербург  Автономная некоммерческая образовательная орга...   
4  Калининградская область  Автономная некоммерческая образовательная орга...   

                   WebSite                                         PathToSoft  \
0          https://raa.ru/  https://raa.ru/sveden/files/Informatika_UPPO_L...   
1  https://nid-design.org/  https://nid-design.org/informatsiya-o-realizue...   
2         https://vepi.ru/  https://vepi.ru/wp-content/uploads/upfiles/sve...   
3        https://eusp.org/  https://eusp.org/sveden/files/Opisanie_PPK_Soc...   
4        https://kiu39.ru/  https://kiu39.ru/_doc/OPOP-KKU/05. 38.02.03 - ...   

                                            SoftInfo  
0                                                NaN  
1                                                NaN  
2  Microsoft Office, Adobe Acrobat, ABBYY FineRea...  
3  Microsoft Office, Email Clients, MMO, Maple, M...  
4  ADOBE ACROBAT READER, MICROSOFT OFFICE WORD, M...

In [27]:
data_w_soft.drop(columns=['Unnamed: 0'], inplace=True)

In [28]:
data_w_soft.head()

ShortName               RegionName  \
0                           РУАН ИМЕНИ Г.Б. МИРЗОЕВА                г. Москва   
1             АНО ВО «НАЦИОНАЛЬНЫЙ ИНСТИТУТ ДИЗАЙНА»                г. Москва   
2  АНОО ВО «Воронежский экономико-правовой инстит...      Воронежская область   
3                                     АНООВО «ЕУСПб»       г. Санкт-Петербург   
4                                       АНООВО «КИУ»  Калининградская область   

                                            FullName                  WebSite  \
0  АВТОНОМНАЯ НЕКОММЕРЧЕСКАЯ ОРГАНИЗАЦИЯ ВЫСШЕГО ...          https://raa.ru/   
1  АВТОНОМНАЯ НЕКОММЕРЧЕСКАЯ ОРГАНИЗАЦИЯ ВЫСШЕГО ...  https://nid-design.org/   
2  Автономная некоммерческая образовательная орга...         https://vepi.ru/   
3  Автономная некоммерческая образовательная орга...        https://eusp.org/   
4  Автономная некоммерческая образовательная орга...        https://kiu39.ru/   

                                          PathToSoft  \
0  https://raa.ru/sveden/files/Informatika_UPPO_L...   
1  https://nid-design.org/informatsiya-o-realizue...   
2  https://vepi.ru/wp-content/uploads/upfiles/sve...   
3  https://eusp.org/sveden/files/Opisanie_PPK_Soc...   
4  https://kiu39.ru/_doc/OPOP-KKU/05. 38.02.03 - ...   

                                            SoftInfo  
0                                                NaN  
1                                                NaN  
2  Microsoft Office, Adobe Acrobat, ABBYY FineRea...  
3  Microsoft Office, Email Clients, MMO, Maple, M...  
4  ADOBE ACROBAT READER, MICROSOFT OFFICE WORD, M...

In [29]:
data_w_soft.dropna(subset=['SoftInfo'], inplace=True)

In [30]:
used_soft_list = data_w_soft['SoftInfo'].to_list()
used_soft_list = list(map(lambda elem: elem.split(','), used_soft_list))

used_soft_list = [elem for elems in used_soft_list for elem in elems]
used_soft_list = list(map(lambda elem: elem.strip(' .'), used_soft_list))

In [31]:
used_soft = pd.Series(used_soft_list)
used_soft.describe()

count                 8995
unique                3088
top       Microsoft Office
freq                   281
dtype: object

In [34]:
unique_soft = used_soft.drop_duplicates()
unique_soft.describe()

count           3088
unique          3088
top       Revit Live
freq               1
dtype: object

In [35]:
unique_soft.head()

0           Microsoft Office
1              Adobe Acrobat
2           ABBYY FineReader
3           Autodesk AutoCAD
4    Microsoft Visual Studio
dtype: object

In [38]:
import_replacement = pd.read_excel('../data/import_replacement.xlsx')
import_replacement.head()

Иностранный продукт                             Отечественный продукт  \
0           1Password                         Менеджер паролей Пассворк   
1                 3CX                            Call-центр Infinity CX   
2                 3CX  IP АТС / Контакт-центр Infinity Эра (Infinity R)   
3                 3CX                             Naumen Contact Center   
4                 3CX                                  VOXDISTRO IP-PBX   

  № в Реестре                               Класс ПО   Класс ПО.1  \
0        6147  Средства аутентификации пользователей          NaN   
1       13330                              Телефония  Колл-центры   
2       19273                    Коммуникационное ПО    Телефония   
3         177                            Колл-центры          NaN   
4        6974                              Телефония          NaN   

    Класс ПО.2 Класс ПО.3 Класс ПО.4 Класс ПО.5 Класс ПО.6 Класс ПО.7  \
0          NaN        NaN        NaN        NaN        NaN        NaN   
1          NaN        NaN        NaN        NaN        NaN        NaN   
2  Колл-центры        NaN        NaN        NaN        NaN        NaN   
3          NaN        NaN        NaN        NaN        NaN        NaN   
4          NaN        NaN        NaN        NaN        NaN        NaN   

  Класс ПО.8 Класс ПО.9  
0        NaN        NaN  
1        NaN        NaN  
2        NaN        NaN  
3        NaN        NaN  
4        NaN        NaN

In [47]:
swsu_soft = pd.read_csv('../data/swsu_soft.csv', delimiter=';')
swsu_soft.head()

soft_name  category   version  \
0            Операционная система "Альт Образование"       NaN       NaN   
1                               Бизнес-курс МАКСИМУМ       NaN  Версия 1   
2                                     Профориентатор       NaN  Стандарт   
3    1С:Предприятие 8.3. Лицензия на сервер (x86-64)       NaN       8.3   
4  1С:Предприятие 8. Клиентская лицензия на 20 ра...       NaN       NaN   

       license  alt_compatible  desk_alt_sp10  serv_alt_sp10  \
0  Proprietary             NaN            NaN            NaN   
1  Proprietary             NaN            NaN            NaN   
2  Proprietary             NaN            NaN            NaN   
3  Proprietary             NaN            NaN            NaN   
4  Proprietary             NaN            NaN            NaN   

   alt_workstation10  alt_education10  alt_server10  ru_replacement  
0                NaN              NaN           NaN             NaN  
1                NaN              NaN           NaN             NaN  
2                NaN              NaN           NaN             NaN  
3                NaN              NaN           NaN             NaN  
4                NaN              NaN           NaN             NaN

In [65]:
swsu_soft['soft_name'] = swsu_soft['soft_name'].str.upper()
swsu_soft.head()

soft_name  category   version  \
0            ОПЕРАЦИОННАЯ СИСТЕМА "АЛЬТ ОБРАЗОВАНИЕ"       NaN       NaN   
1                               БИЗНЕС-КУРС МАКСИМУМ       NaN  Версия 1   
2                                     ПРОФОРИЕНТАТОР       NaN  Стандарт   
3    1С:ПРЕДПРИЯТИЕ 8.3. ЛИЦЕНЗИЯ НА СЕРВЕР (X86-64)       NaN       8.3   
4  1С:ПРЕДПРИЯТИЕ 8. КЛИЕНТСКАЯ ЛИЦЕНЗИЯ НА 20 РА...       NaN       NaN   

       license  alt_compatible  desk_alt_sp10  serv_alt_sp10  \
0  Proprietary             NaN            NaN            NaN   
1  Proprietary             NaN            NaN            NaN   
2  Proprietary             NaN            NaN            NaN   
3  Proprietary             NaN            NaN            NaN   
4  Proprietary             NaN            NaN            NaN   

   alt_workstation10  alt_education10  alt_server10  ru_replacement  
0                NaN              NaN           NaN             NaN  
1                NaN              NaN           NaN             NaN  
2                NaN              NaN           NaN             NaN  
3                NaN              NaN           NaN             NaN  
4                NaN              NaN           NaN             NaN

In [66]:
import_replacement['Иностранный продукт'] = import_replacement['Иностранный продукт'].str.upper()
import_replacement['Отечественный продукт'] = import_replacement['Отечественный продукт'].str.upper()
import_replacement.head()

Иностранный продукт                             Отечественный продукт  \
0           1PASSWORD                         МЕНЕДЖЕР ПАРОЛЕЙ ПАССВОРК   
1                 3CX                            CALL-ЦЕНТР INFINITY CX   
2                 3CX  IP АТС / КОНТАКТ-ЦЕНТР INFINITY ЭРА (INFINITY R)   
3                 3CX                             NAUMEN CONTACT CENTER   
4                 3CX                                  VOXDISTRO IP-PBX   

  № в Реестре                               Класс ПО   Класс ПО.1  \
0        6147  Средства аутентификации пользователей          NaN   
1       13330                              Телефония  Колл-центры   
2       19273                    Коммуникационное ПО    Телефония   
3         177                            Колл-центры          NaN   
4        6974                              Телефония          NaN   

    Класс ПО.2 Класс ПО.3 Класс ПО.4 Класс ПО.5 Класс ПО.6 Класс ПО.7  \
0          NaN        NaN        NaN        NaN        NaN        NaN   
1          NaN        NaN        NaN        NaN        NaN        NaN   
2  Колл-центры        NaN        NaN        NaN        NaN        NaN   
3          NaN        NaN        NaN        NaN        NaN        NaN   
4          NaN        NaN        NaN        NaN        NaN        NaN   

  Класс ПО.8 Класс ПО.9  
0        NaN        NaN  
1        NaN        NaN  
2        NaN        NaN  
3        NaN        NaN  
4        NaN        NaN

In [82]:
swsu_soft_list = swsu_soft['soft_name'].to_list()

In [83]:
swsu_soft_list

['ОПЕРАЦИОННАЯ СИСТЕМА "АЛЬТ ОБРАЗОВАНИЕ"',
 'БИЗНЕС-КУРС МАКСИМУМ',
 'ПРОФОРИЕНТАТОР',
 '1С:ПРЕДПРИЯТИЕ 8.3. ЛИЦЕНЗИЯ НА СЕРВЕР (X86-64)',
 '1С:ПРЕДПРИЯТИЕ 8. КЛИЕНТСКАЯ ЛИЦЕНЗИЯ НА 20 РАБОЧИХ МЕСТ',
 '1С:ПРЕДПРИЯТИЕ 8. КЛИЕНТСКАЯ ЛИЦЕНЗИЯ НА 100 РАБОЧИХ МЕСТ',
 '1С:УЧЕТ ОБРАЩЕНИЙ ПРОФ. ЭЛЕКТРОННАЯ ПОСТАВКА',
 '1С:ЭЛЕКТРОННОЕ ОБУЧЕНИЕ. КОНСТРУКТОР КУРСОВ ПРОФ',
 '1С:ПРЕДПР.8.КОМПЛЕКТ ДЛЯ ОБУЧЕНИЯ В ВЫСШИХ И СРЕДНИХ УЧЕБНЫХ ЗАВЕДЕНИЯХ (USB)',
 'МАТЕРИАЛЫ СЛУШАТЕЛЯ СЕРТИФИЦИРОВАННОГО КУРСА "1С:ПРЕДПРИЯТИЕ 8. ОСНОВНЫЕ ОБЪЕКТЫ" ВЕРСИЯ 8.2',
 'КОМПЛЕКТ МАТЕРИАЛОВ ПЕРПОДАВАТЕЛЯ СЕРТИФИЦИРОВАННОГО КУРСА "1С:ПРЕДПРИЯТИЕ 8. БУХГАЛТЕРСКИЕ ЗАДАЧИ"',
 'КОМПЛЕКТ МАТЕРИАЛОВ ПЕРПОДАВАТЕЛЯ СЕРТИФИЦИРОВАННОГО КУРСА "1С:ПРЕДПРИЯТИЕ 8. ОПЕРАТИВНЫЕ ЗАДАЧИ"',
 'КОМПЛЕКТ МАТЕРИАЛОВ ПЕРПОДАВАТЕЛЯ СЕРТИФИЦИРОВАННОГО КУРСА "ОСНОВЫ ПРОГРАММИРОВАНИЯ В "1С:ПРЕДПРИЯТИЕ 8" ДЛЯ ШКОЛЬНИКОВ"',
 'МАТЕРИАЛЫ СЛУШАТЕЛЯ СЕРТИФИЦИРОВАННОГО КУРСА "ОСНОВЫ ПРОГРАММИРОВАНИЯ В "1С:ПРЕДПРИЯТИЕ 8" ДЛЯ ШКОЛЬНИКОВ" МОДУЛЬ 1',
 'КОН